packages/playwright-ct-react/registerSource.mjs

```diff
@@ -77,17 +79,39 @@ window.playwrightMount = async (component, rootElement, hooksConfig) => {
       App = () => wrapper;
   }
 
-  ReactDOM.render(App(), rootElement);
+  if (rootRegistry.has(rootElement)) {
+    throw new Error(
+        'Attempting to mount a component into an container that already has a React root. ' +
+        'Either unmount the React root first with `playwrightUnmount(rootElement)` or update the root instead with `playwrightUpdate(rootElement, component)`'
```

Comment:
These are internal methods, I don't think user will be able to act given this information.

Reply:
Gotcha, so I just leave a "This is a bug" in the error message? Or do you know if/how a user can hit this codepath?

Reply:
I think "Attempting to mount a component into an container that already has a React root." is good enough to be actionable. Just don't point to the internal methods.

Reply:
Done